tests/pytests/test_debug_commands.py

```diff
@@ -221,8 +215,8 @@ def testTTLcommands(self):
     def testStopAndResumeWorkersPool(self):
         if not MT_BUILD:
             self.env.skip()
-        self.env.expect('FT.DEBUG', 'WORKER_THREADS').error().contains("wrong number of arguments for"
-                                                                              " 'FT.DEBUG' command")
+        self.env.expect('FT.DEBUG', 'WORKER_THREADS').error().contains(
+            "wrong number of arguments for 'FT.DEBUG|WORKER_THREADS' command")
```

Comment:
```suggestion
            "wrong number of arguments for 'FT.DEBUG WORKER_THREADS' subcommand")
```
(the pipe looked strange to me...)

Reply:
It’s redis error message LOL
I can change the test, but it will fail…